jdvalera/servo_example_modified.ino

```diff
@@ -0,0 +1,44 @@
+#include <Servo.h>  // Include the Servo library
+
+Servo servoVarName;  // Declare a variable as a Servo object
+int servoPin = 2;  // Declare and initialize a variable to hold the servo's pin
+
+// The setup() function runs once at the startup
+void setup() {
+  // Attach the pin to the servo variable in the setup
+  servoVarName.attach(servoPin);
+}
+
+// The loop() function runs after the startup and runs indefinitely
+void loop() {
+  // Moves servo from one extreme to another
+  for (int pos = 0; pos < 180; pos+=10) {
```

Comment:
Could you fix your whitespace here on line 15?

`for (int pos = 0; pos < 180; pos+=10)`
should look like
`for (int pos = 0; pos < 180; pos += 10)`

I know this is a bit anal, but code clarity and readability is important.